pkg/controller/plan/kubevirt.go

```diff
@@ -235,27 +266,29 @@ func (r *KubeVirt) DeleteDataVolumes(vm *plan.VMStatus) (err error) {
 	return
 }
 
-// Delete the importer pod for a PersistentVolumeClaim.
-func (r *KubeVirt) DeleteImporterPod(pvc core.PersistentVolumeClaim) (err error) {
-	var pod *core.Pod
+// Delete the importer pods for a PersistentVolumeClaim.
+func (r *KubeVirt) DeleteImporterPods(pvc core.PersistentVolumeClaim) (err error) {
+	var pods []core.Pod
 	var found bool
-	pod, found, err = r.GetImporterPod(pvc)
+	pods, found, err = r.GetImporterPods(pvc)
 	if err != nil || !found {
 		return
 	}
-	err = r.Destination.Client.Delete(context.TODO(), pod)
-	if err != nil {
-		err = liberr.Wrap(err)
-		return
+	for _, pod := range pods {
+		err = r.Destination.Client.Delete(context.TODO(), &pod)
+		if err != nil {
+			err = liberr.Wrap(err)
+			return
```

Comment:
I don't see a scenario that we'll fail to remove only one of the pods but wouldn't it be better to try to proceed with removing the other pods?